spanner/spansql/parser_test.go

```diff
@@ -114,6 +114,20 @@ func TestParseQuery(t *testing.T) {
 				},
 			},
 		},
+		// with table hint
+		{`SELECT * FROM Packages@{FORCE_INDEX=PackagesIdx} WHERE package_idx=@packageIdx`,
```

Comment:
nit: add an additional test that verifies that the parser also accepts multiple table hints.